selfdrive/car/hyundai/values.py

```diff
@@ -1327,6 +1333,7 @@ class Buttons:
       b'\xf1\x00CV1 MFC  AT USA LHD 1.00 1.05 99210-CV000 211027',
       b'\xf1\x00CV1 MFC  AT EUR LHD 1.00 1.05 99210-CV000 211027',
     ],
+    (Ecu.adas, 0x730, None): [],
```

Comment:
make sure this isn't an essential ECU. only HDA2 cars have an ADAS ECU, though perhaps the camera will respond to this query

Reply:
Yep, checked and it still exact matches without